src/pull_automerge/core.clj

```diff
@@ -24,16 +24,15 @@
     }
   })
 
-(defn get-opened-labelled-issues [org repo label options]
+(defn get-opened-labelled-issues [org repo options & labels]
   (def pulls-result @(http/get 
     (str "https://api.github.com/repos/" org "/" repo "/issues"
-      "?labels=" label
+      "?labels=" (apply str(interpose "," labels))
       "&state=open"
-      "&page=0&per_page=1"
       "&sort=created&direction=asc")
     options))
   (println "Retrieve automerge issues status: " (pulls-result :status))
-  (parse-string (pulls-result :body) true))
+  pulls-result)
```

Comment:
I was having trouble turning it from the string into a map later on to look for other labels so I'm just returning the json response. There's probably a better way to do this.

Reply:
It's a list of objects/maps (each object/map being a github issue). It's not a string. You should be able to parse it here as I don't think there is a use case for the string version of the body.